src/controllers/user.js

```diff
@@ -80,3 +80,15 @@ export const updateById = async (req, res) => {
     return sendMessageResponse(res, 500, 'Unable to update user')
   }
 }
+
+export const updateUserCohortById = async (req, res) => {
+  const { id, cohort_id: cohortId } = req.body
```

Comment:
Rather than getting the userId from the request body, could you get it from request params instead since it's part of the route?

Reply:
I could get the cohortId from the route params. I am not sure how to get the userId from it?

Reply:
Acutally, I just realised my error. Will change this.